src/main/haskell/kore/src/Kore/Unification/UnifierImpl.hs

```diff
@@ -197,13 +200,14 @@ solveGroupedSubstitution tools substitutionSimplifier var patterns = do
         ( Predicated
             { term = ()
             , predicate = ExpandedPattern.predicate predSubst
-            , substitution = termAndSubstitution predSubst
+            , substitution = Substitution.wrap $ termAndSubstitution predSubst
             }
         , proof
         )
   where
     termAndSubstitution s =
-        (var, ExpandedPattern.term s) : ExpandedPattern.substitution s
+        (var, ExpandedPattern.term s) :
```

Comment:
I think that the style guide requires the operator to be on the next line. I may be wrong, but if you're not moving the operator you should probably indent the next line.